packages/cypress/src/plugins/plugin.ts

```diff
@@ -183,6 +184,7 @@ async function buildCypressTargets(
   options: CypressPluginOptions,
   context: CreateNodesContext
 ): Promise<CypressTargets> {
+  const pmc = getPackageManagerCommand();
```

Comment:
We need to be careful with this function. It spawns a function to check the version of some package managers.

Can you please move this higher so that we only call it once per plugin per graph creation?

In it's current spot, it will happen once per config file.